### PR TITLE
PyCharm 2019.3.1 easyconfig

### PR DESCRIPTION
**Pull request by shahzebsiddiqui**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/PyCharm/PyCharm-2019.3.1.eb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+version = "2019.3.1"
+
+homepage = 'https://www.jetbrains.com/pycharm/'
+description = """PyCharm Community Edition: Python IDE for Professional Developers"""
+
+toolchain = SYSTEM
+
+source_urls = ['https://download-cf.jetbrains.com/python']
+sources = ['pycharm-community-%s.tar.gz' % version]
+checksums = []
```
Comment (boegel) [inline easybuild/easyconfigs/p/PyCharm/PyCharm-2019.3.1.eb]: @shahzebsiddiqui Please include a SHA256 checksum, and use `%(version)s`:

```python
sources = ['pycharm-community-%(version)s.tar.gz']

Comment (shahzebsiddiqui) [reply easybuild/easyconfigs/p/PyCharm/PyCharm-2019.3.1.eb]: i made the changes. Hope all is good now.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3105.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/078be34ef40450854beba19bcc988cd3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/cfdbf4478da57002fda10b0b0402da5a for a full test report.

**Comment by boegel** [comment]
Going in, thanks @shahzebsiddiqui!

**Pull request merged**